src/SourceNodes/sourcenode.js

```diff
@@ -384,8 +388,7 @@ class SourceNode extends GraphNode {
             this._state = STATE.playing;
         }
 
-        if (currentTime >= this._stopTime) {
-            clearTexture(this._gl, this._texture);
+        if (currentTime > this._stopTime) {
```

Comment:
is there a reason why you've changed from `>=` to `>` only?

Reply:
good catch, I actually forgot to put it back